docs/specline.rst

```diff
@@ -35,18 +35,21 @@ The base attributes for the SpectralLine class are:
 ================ ================= ========== ========================================
 Property         Variable          Type       Description
 ================ ================= ========== ========================================
+Limits           limits            LineLimits The redshift and limits of the line in redshift, velocity
+                                              (w/r to its redshift) and observed wavelength.
 RA, Dec          attrib['coord']   Coord      astropy.coordinate
-Redshift         attrib['z']       float      Reference redshift
-Redshift sigma   attrib['sig_z']   float      Reference redshift uncertainty
```

Comment:
I expect this will happen and should, but I decided I'd rather
not make it a default attribute.

As for name, z_measured might be best.